.github/workflows/deploy.yml

```diff
@@ -0,0 +1,12 @@
+name: Deploy to Cloudflare Pages
+
+on:
+  workflow_run:
+    workflows: ["Build"]
+    types:
+      - completed
+
+jobs:
+  deploy-to-cloudflare:
+    uses: ubiquity/.github/.github/workflows/deploy.yml@main
```

Comment:
This should be in `@ubiquity/cloudflare-deploy-action`

Reply:
No, it calls a [reusable deploy workflow](https://github.com/ubiquity/.github/blob/main/.github/workflows/deploy.yml) which in turn calls a [reusable deploy action](https://github.com/ubiquity/.github/blob/585e4aa9840c565dcaba4e6777ec23b6c474f427/.github/workflows/deploy.yml#L43)

Reply:
Updated https://github.com/ubiquity/ts-template/pull/37/commits/90671efbc95e64927689d5c8087eac1bc4a4e633